LocationSync/LocationSync.cpp

```diff
@@ -27,6 +27,25 @@ namespace Plugin {
     static Core::ProxyPoolType<Web::Response> responseFactory(4);
     static Core::ProxyPoolType<Web::JSONBodyType<LocationSync::Data>> jsonResponseFactory(4);
 
+    namespace {
+        const string TZ_FILE = "/opt/persistent/timeZoneDST";
```

Comment:
This looks like a magicals. Its good programming practice to avoid magicals and I think to achieve backwards compatibility, you can easily avoid it if you put it in the plugin config, see next remark.